src/app/loadout/loadout-edit/LoadoutEditBucketDropTarget.m.scss

```diff
@@ -29,4 +29,5 @@
 // items entirely.
 .dragOver {
   pointer-events: none;
+  opacity: 0;
```

Comment:
This just hides the current section when we are showing the equipped/unequipped drop targets. It looked really weird that you could see everything underneath the two button things. I added a part to the video where to show what happens when you drap something like armour

Reply:
definitely prefer it without the ghostly icons & borders, yeah

Reply:
```suggestion
  visibility: hidden;
```

Reply:
(you can probably remove the pointer-events too)

Reply:
I will pull it down and see if I can remove it

Reply:
Done